src/bmi.jl

```diff
@@ -125,58 +140,82 @@ function BMI.get_output_var_names(model::Model)
 end
 
 function BMI.get_var_grid(model::Model, name::String)
```

Comment:
Not for this PR, but I often prefer to add return type annotations for readability, `BMI.get_var_grid(model::Model, name::String)::Int`.